docker-compose.yml

```diff
@@ -60,6 +60,7 @@ services:
       PRIVATE_KEY: ${PRIVATE_KEY}
     ports:
       - 127.0.0.1:5000:5000
+      - 127.0.0.1:80:80
```

Comment:
we need that because at some point the browser stops talking to the web app and talks directly to the Dela node URL (i.e. the proxy server deployed in the backend)